authentik/core/tests/test_groups_api.py

```diff
@@ -144,10 +157,10 @@ def test_superuser_update_no_perm(self):
             reverse("authentik_api:group-detail", kwargs={"pk": group.pk}),
             data={"is_superuser": False},
         )
-        self.assertEqual(res.status_code, 400)
+        self.assertEqual(res.status_code, 403)
         self.assertJSONEqual(
             res.content,
-            {"is_superuser": ["User does not have permission to set superuser status to False."]},
+            {"detail": "User does not have permission to set the given superuser status."},
```

Comment:
The only thing I dislike about this change is that we no longer get the error tied to the specific `is_superuser` field but rather just a general error